validator_worker/src/lib.rs

```diff
@@ -52,19 +52,19 @@ mod test {
     use adapter::DummyAdapter;
     use primitives::adapter::AdapterOptions;
     use primitives::config::configuration;
-    use primitives::util::tests::prep_db::{DUMMY_CHANNEL, IDS};
+    use primitives::util::tests::prep_db::{AUTH, DUMMY_CHANNEL, IDS};
     use primitives::{BalancesMap, Channel};
 
     fn setup_iface(channel: &Channel) -> SentryApi<DummyAdapter> {
         let adapter_options = AdapterOptions {
             dummy_identity: Some(IDS["leader"].clone()),
```

Comment:
you did change the `IDS` to also have `String` for key and I am not sure how this is working tbh ?! :thinking: